auth-api/src/auth_api/resources/bulk_user.py

```diff
@@ -37,7 +37,6 @@ class BulkUser(Resource):
     @staticmethod
     @TRACER.trace()
     @cors.crossdomain(origin='*')
```

Comment:
Any reason to make this endpoint public ?

Reply:
missed it..Will add..thanks for the catch

Reply:
I wil do it in the next PR..will add some tests as well..Merging this